src/renderer/components/CustomEdge/CustomEdge.tsx

```diff
@@ -92,67 +92,42 @@ export const CustomEdge = memo(
                 onMouseOver={() => hoverTimeout()}
             >
                 <path
-                    className="edge-chain-links"
+                    className={useMemo(
```

Comment:
Why did you memo this? You're just creating a string, it's not expensive. In fact, I even suspect that the overhead of the memo + the function + deps array is more expensive than the string interpolation...

Reply:
It at least appeared to improve performance... Can't really explain it but dragging nodes around was laggy before i did this

Reply:
That's strange cause it really shouldn't....

Reply:
it apparently doesnt anymore, so might have been a coincidence